usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -137,8 +137,7 @@ LEFT JOIN (
        )
      ) treasury_accounts
  FROM
-   federal_account fa
```

Comment:
Was this table helping to reduce duplicate/invalid values?

Reply:
Speaking to only the transaction_delta_view, I believe that was a result of a bad copy and paste on my part between Federal Account and TAS.